src/Cdn77/ruleset.xml

```diff
@@ -46,19 +46,19 @@
                 public properties,
                 protected properties,
                 private properties,
-                public static abstract methods,
-                public static methods,
-                protected static abstract methods,
-                protected static methods,
-                private static methods,
                 constructor,
                 static constructors,
                 destructor,
                 magic methods,
+                public static abstract methods,
                 public abstract methods,
+                public static methods,
```

Comment:
prohodil bych 54 a 55

Reply:
why?

Reply:
let's keep contract first

Reply:
it's just like my opinion but for me this order seems logical.. keeping static together and non-static together. I know we have tools for this, but I think this order is quite straight forward, when "trying to figure out" where to place the method.

public/protected/private static abstract
public/protected/private static

public/protected/private abstract
public/protected/private

but I can live with both versions for sure.

Reply:
So we keep contract (abstract) first, right?

Reply:
yeah, but grouped as they were until now basically.